content/docs/user-guide/gems/reference/environment/vegetation.md

```diff
@@ -0,0 +1,207 @@
+|     | **Rotation Min X** | Override value for the minimum rotation offset on the X axis. | Float |
+|     | **Rotation Max X** | Override value for the maximum rotation offset on the X axis. | Float |
+|     | **Rotation Min Y** | Override value for the minimum rotation offset on the Y axis. | Float |
+|     | **Rotation Max Y** | Override value for the maximum rotation offset on the Y axis. | Float |
+|     | **Rotation Min Z** | Override value for the minimum rotation offset on the Z axis. | Float |
+|     | **Rotation Max Z** | Override value for the maximum rotation offset on the Z axis. | Float |
+| Scale |     | Settings for controlling per instance overrides for the scale modifier component. |     |
+|     | **Scale Override Enabled** | When enabled, the scale modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Scale Min** | Override value for the minimum scale multiplier. | Float |
+|     | **Scale Max** | Override value for the maximum scale multiplier. | Float |
+| Altitude Filter |     | Settings for controlling per instance overrides for the altitude filter component. |     |
+|     | **Altitude Filter Override Enabled** | When enabled, the altitude filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Altitude Filter Min** | Override value for the minimum altitude accepted by the filter. | Float |
+|     | **Altitude Filter Max** | Override value for the maximum altitude accepted by the filter. | Float |
+| Distance Between Filter (Radius) |     | Settings for controlling per instance overrides for the distance between filter component. |     |
+|     | **Distance Between Filter Override Enabled** | When enabled, the distance between filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Bound Mode** | When performing distance checks between two instances, this setting determines whether the radius of the mesh asset is used vs a manually entered radius<br><br>Radius<br><br>MeshRadius. |     |
+|     | **Radius Min** | User defined radius for distance checks. | Float |
+| Surface Slope Alignment |     |     |     |
+|     | **Surface Slope Alignment Override Enabled** | When enabled, the slope alignment modifier component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
+|     | **Surface Slope Alignment Min** | Override minimum alignment strength. | Float |
+|     | **Surface Slope Alignment Max** | Override maximum alignment strength. | Float |
+| Slope Filter |     |     |     |
+|     | **Slope Filter Override Enabled** | When enabled, the slope filter component can use the per instance override values specified in the descriptor instead of the component configuration. | Bool |
```

Comment:
```suggestion
|     | **Slope Filter Override Enabled** | When enabled, the [Slope Filter](/docs/user-guide/components/reference/vegetation-filters/vegetation-slope-filter) component can use the per-instance override values specified in the descriptor instead of the component configuration. | Bool |
```